packages/livechat/.storybook/main.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
kinda feels like maybe this folder isn't supposed to be checked in?  Not sure

Reply:
seems it was part of moving that to the monorepo.

For the record feels like this shouldn't have been in a patch release.  As isn't a fix on its own.  It might have included fixes.. but at this point it should have just gone to 4.8.x which hits feature freeze soon

Reply:
I think there is a fix done **after** moving to monorepo, that's why.. @MartinSchoeler can probably clarify though